hr_course/i18n/es.po

```diff
@@ -24,6 +24,7 @@ msgstr "Ausente"
 
 #. module: hr_course
 #: model:ir.model.fields,field_description:hr_course.field_hr_course__message_needaction
+#: model:ir.model.fields,field_description:hr_course.field_hr_course_schedule__message_needaction
```

Comment:
you cannot add translation to already introduced addon as it may cause issues during merge if in meanwhile it was updated with weblate, so for translations we need merge work first then update translation through weblate